composables/collectionActivity/helpers.ts

```diff
@@ -92,7 +93,19 @@ const updateOwnerWithNewNft = ({
     lastestTimeStamp > owner.lastActivityTimestamp
       ? lastestTimeStamp
       : owner.lastActivityTimestamp
+
   owner.nfts = [...owner.nfts, nft]
+
+  let longestHoldTimestamp = lastestTimeStamp
+
+  owner.nfts.forEach((nft) => {
+    longestHoldTimestamp = Math.max(
+      longestHoldTimestamp,
+      new Date(nft.updatedAt).getTime(),
```

Comment:
Is this time correct? If user relists the nft with a new price, the `updatedAt` may change. It does not represent holding time.

Reply:
Thanks for showing an example. You're right.